package.json

```diff
@@ -25,7 +25,9 @@
   },
   "dependencies": {
     "bluebird": "^3.1.1",
-    "chalk": "^1.1.1",
-    "vueify": "^8.3.0"
+    "chalk": "^1.1.1"
+  },
+  "peerDependencies": {
+    "vueify": "*"
```

Comment:
I'm of two minds. This gives the most flexibility, but it also could lead to thing breaking if the API changes in the future. I really dislike `*` dependencies, but vueify _probably_ won't change its API...

Reply:
Your concerns are totally valid. A better solution would be "~9.4.1" but this puts the onus on the package mainter, whereas "*" puts it on the package user. Since the package is targeted at developers I didn't think there would be much risk in doing so.